developer_setup.md

```diff
@@ -155,9 +155,12 @@ git fetch other_user
 ### Get the Rails environment up and running
 
 ```bash
-bin/setup
+bin/setup                  # Installs dependencies, config, prepares database, etc
+bundle exec rake evm:start # Starts the ManageIQ EVM Application in the background
```

Comment:
Hmm I'm not sure what you mean here; haven't dug in enough to know about those options or what's available. Is this beyond the minimal mode guide referenced below?
I'll add whatever specific things you think would be helpful.